### ISSUE TITLE
Bug: eslint 报错以及 i18n-ally 不能正确找到语言文件问题 

### ISSUE THREAD

**Issue by elkon028** [vbenjs/vue-vben-admin#4211]
### Version

Vben Admin V5

### Describe the bug?

因为 `.vscode/settings.json` 缺少`editor.defaultFormatter`和`i18n-ally.pathMatcher`
造成 eslint 报错 以及 i18n-ally 不能正确找到语言文件问题

- 用户 vscode 的 以下配置可能与 vben-admin 项目 vscode 配置不适应
- 所以需要在 `.vscode/settings.json` 里添加配置：
```json
// 格式化程序指定为 prettier
"[vue]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode"
  },
"[jsonc]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode"
  },
 "[json]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode"
  },
 "[***其它***]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode"
  },
// 配置 i18n-ally
"i18n-ally.pathMatcher": "{locale}.json",
```

### Reproduction

<img width="1487" alt="image" src="https://github.com/user-attachments/assets/269ae0cf-544c-4b26-a4b7-ad628e4c832e">

<img width="1321" alt="image" src="https://github.com/user-attachments/assets/225d8fe7-79e0-4955-9e26-3b78e12d77a1">

### System Info

```shell
System:
    OS: macOS 14.6.1
    CPU: (14) arm64 Apple M3 Max
    Memory: 1.11 GB / 36.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.16.0 - /usr/local/bin/node
    npm: 10.8.2 - /usr/local/bin/npm
    pnpm: 9.7.1 - /usr/local/bin/pnpm
    Watchman: 2024.01.22.00 - /opt/homebrew/bin/watchman
  Browsers:
    Chrome: 127.0.6533.120
    Safari: 17.6
```

### Validations

- [X] Read the [docs](https://anncwb.github.io/vue-vben-admin-doc/)
- [X] Ensure the code is up to date. (Some issues have been fixed in the latest version)
- [X] I have searched the [existing issues](https://github.com/vbenjs/vue-vben-admin/issues) and checked that my issue does not duplicate any existing issues.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vbenjs/vue-vben-admin/discussions) or join our [Discord Chat Server](https://discord.gg/8GuAdwDhj6).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by forose** [comment]
我发现有playground的时候，在web-antd里是可以找到对应的中英文文件的，但是如果没有的话 就找不到，这个时候默认找的是packages里面的了。

**Comment by anncwb** [comment]
fixed #4240
